### PR TITLE
feat: add S3 source to CLI

### PR DESCRIPTION
**Pull request by tpluscode**

### CODE REVIEW

File: packages/cli/README.md [hunk 1]
```diff
@@ -29,6 +29,18 @@ Loads input files a local filesystem path.
 * `sourceDir`
   * default: `/input`
 
+### `--from s3`
+
+Loads input files from an S3 bucket.
+
+#### Arguments
+
+* `S3-ENDPOINT`
```
Comment (martinmaillard) [inline packages/cli/README.md]: Any reason to have these in all-caps?

File: packages/cli/pipelines/from-s3.ttl [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+<#LoadCsv>
+  :steps [ :stepList ( _:openCsvFromBucket ) ] ;
+  :variables [ :variable
+    [ a :Variable; :name "S3-ENDPOINT" ] ,
+    [ a :Variable; :name "S3-BUCKET" ]
+  ].
+
+_:openCsvFromBucket
+  a                  :Step ;
```
Comment (martinmaillard) [inline packages/cli/pipelines/from-s3.ttl]: Sorry, I'm not going to nit-pick on this every time, promise. But I don't understand how this formatting works. What are these huge spaces? 

The `LoadCsv` part looks good, but the `openCsvFromBucket` part looks totally different...

File: packages/cli/README.md (hunk 1 above)
Comment (tpluscode) [reply packages/cli/README.md]: Hm, no reason. Copied from env variables the API uses. Lower case would be more consistent?

Comment (martinmaillard) [reply packages/cli/README.md]: Since all the other arguments are lowercase, I would say yes

File: packages/cli/pipelines/from-s3.ttl (hunk 2 above)
Comment (tpluscode) [reply packages/cli/pipelines/from-s3.ttl]: Haha, sure, I was confused too initially.

So you know how RDF is `S P O` triples. The formatting aligns each of those parts vertically. The large gap is to match the space `code:implementedBy` takes on the next line. Same below

```
code:implementedBy [ code:link <file:../lib/s3#openFile> ;
                     a         code:EcmaScript ] ;
                     ^         ^
```

Comment (martinmaillard) [reply packages/cli/pipelines/from-s3.ttl]: Ok, not sure I like it. As long as it's consistent, I can get used to it.

**Pull request merged**